surveymonkey/api_surveymonkey.py

```diff
@@ -86,22 +98,40 @@ def get_surveys(self, **kwargs):
         """
         Returns a list of surveys owned or shared with the authenticated user.
         """
+        cache_key = "{}-{}-{}".format(SURVEY_MONKEY_API_TAG, "all_surveys", self.client_id)
+        all_surveys_data = cache.get(cache_key)
+
+        if self.surveymonkey_api_cache_duration == 0:
+            cache.delete(cache_key)
+        elif all_surveys_data:
```

Comment:
Remove the lines 104 and 105
```suggestion
        if all_surveys_data and not self.surveymonkey_api_cache_duration:
            return all_surveys_data
```